content.test.ts

```diff
@@ -238,3 +238,43 @@ test('e2e content without any catalogs', { timeout: 120000 }, async () => {
   expect(content.version).toBe(0);
   expect(content.commands.length).toBe(1);
 });
+
+test('list content', { timeout: 120000 }, async () => {
+  client.configureOrg({
+    companyName: "Cortex Click",
+    companyInfo: "Cortex Click provides an AI platform for go-to-market. Cortex click allows you to index your enterprise knowledge base, and create agents called Cortexes that automate sales and marketing processes like SEO, content writing, RFP generation, customer support, sales document genearation such as security questionairres and more.",
+    personality: ["friendly and helpful", "expert sales and marketing professional", "experienced software developer"],
+    rules: ["never say anything disparaging about AI or LLMs", "do not offer discounts"],
+  })
+
+  const cortex = await client.configureCortex(`cortex-list-content-test`, {
+    friendlyName: "Cortex AI",
+    instructions: ["answer questions about the cortex click AI GTM platform"],
+    public: true,
+  });
+
+  // create content
+  const title = "Overview of the Cortex Click AI GTM Platform";
+  const prompt = "Write a blog post about the Cortex Click AI GTM Platform. Elaborate on scenarios, customers, and appropriate verticals. Make sure to mention the impact that AI can have on sales and marketing teams."
+  const content = await cortex.generateContent({ title, prompt });
```

Comment:
Do you think we need to generate new content for this test? 

I wonder if we should add the logic for this test as a part of the basic e2e generate content test at the top of the file? Would save us 120s plus the cost of the LLM calls. 

Totally understand that this is cleaner, but wondering if it is worth compromising due to the practical limitations

Reply:
Yeah we could do that. I wanted the test to encapsulate it's requirements, but yeah I could throw it at the end of that test instead for practical reasons.